packages/analytics/dashboard-renderer/src/components/DashboardRenderer.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="kong-ui-public-dashboard-renderer">
+    <KAlert
+      v-if="!queryBridge"
+      appearance="danger"
+    >
+      No query bridge provided.  Unable to render dashboard.
```

Comment:
issue: these strings should be moved to the `en.json` file

Reply:
Done.